<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalOrderForm</class>
 <widget class="QWidget" name="NormalOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>600</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page_7">
    <widget class="QTableWidget" name="beverageWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QTableWidget" name="snackWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QTableWidget" name="instantWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_10">
    <widget class="QTableWidget" name="milkiceWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_11">
    <widget class="QTableWidget" name="drinkWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_12">
    <widget class="QTableWidget" name="neceWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_13">
    <widget class="QTableWidget" name="allWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>170</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="beveraageTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>음료</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="snackTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>스낵</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="instantTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>즉석식품</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="mikiceTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>유제품/아이스크림</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="drinkTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>주류</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="neceTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>생활용품</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="allTool_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <attribute name="label">
     <string>전체</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTableWidget" name="orderWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
